libkbfs/keybase_daemon_rpc_test.go

```diff
@@ -92,10 +92,10 @@ func (c *fakeKeybaseClient) Call(ctx context.Context, s string, args interface{}
 			return fmt.Errorf("Could not find user info for UID %s", uid)
 		}
 
-		*res.(*keybase1.Identify2Res) = keybase1.Identify2Res{
-			Upk: keybase1.UserPlusKeys{
-				Uid:      uid,
-				Username: string(userInfo.Name),
+		*res.(*keybase1.IdentifyLiteRes) = keybase1.IdentifyLiteRes{
+			Ul: keybase1.UserOrTeamLite{
```

Comment:
It was supposed to be symmetric with the field `Identify2Res.Upk` ("UserPlusKeys"), but it stems from a previous draft of the protocol when I called the type just "UserLite".  Sorry, would need to change it in the client protocol (and break everyone's CIs) to fix it.